### DISCUSSION TITLE
Connection issues

### DISCUSSION THREAD

**Discussion by ALTrijssenaar** [actions/virtual-environments#6053] [General questions]
When I try to rebuild the ubuntu20-04 image on an Azure Pipeline Agent for recreating a private hosted Virtual Machine Scale Set based on the Microsoft provided images the packer process keeps on failing on all different kinds of network and/or resource related issues.

- Connection timed out to http://azure.archive.ubuntu.com
- Cleanup.sh: Error uploading script: Error copying input data into local temporary file. Check that TEMPDIR has enough space. Please see https://www.packer.io/docs/other/environment-variables#tmpdirfor more info. Error: connection reset
 
Anyone else noticed these problems? 
Is there a way to prevent that? 
How is Microsoft dealing with those problems?

TAG: ubuntu20/20220807.1

**Comment by al-cheb** [comment]
Anyone else noticed these problems? - Quite often for http://azure.archive.ubuntu.com/ repository
Is there a way to prevent that? - Retries only and try centralus, westus2 locations
How is Microsoft dealing with those problems? - Retries or escalate to repo maintainers(canonical.com)
